magento/magento2
# Fix import iframe visibility

### Description
An iframe is displayed at the bottom of the import page during validation and subsequent data import

The `display` attribute with the value `none` is added to the iframe, previously it was `style="display:none"`, but during refactoring this was incorrectly changed.

### Related Commit
https://github.com/magento/magento2/commit/1edf22403419c04d09ccff1c587ad2479b524d37#diff-3160ffecaaf7ab21ba68a3b535d34ab5719602af66952aabe8a821cfac42da9aR172

### Manual testing scenarios (*)
1. Go to the import page in the admin panel
2. Select any type of entity
3. Upload the file for import
4. Click "Check data"
5. Click "Import"
6. Scroll window to the bottom


![image](https://github.com/user-attachments/assets/16c6fef3-6fb2-4ff0-9841-4bc5a5ab6183)


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#39267: Fix import iframe visibility
